doc/api/cli.md

```diff
@@ -624,6 +624,23 @@ Preload the specified module at startup.
 Follows `require()`'s module resolution
 rules. `module` may be either a path to a file, or a node module name.
 
+### `--unhandled-rejections=mode`
```

Comment:
It seems this should be placed before `` ### `--use-bundled-ca`, `--use-openssl-ca` `` section.